mavros_msgs/msg/CommandCode.msg

```diff
@@ -1,37 +1,206 @@
-# Some MAV_CMD command codes.
+# MAV_CMD command codes.
 # Actual meaning and params you may find in MAVLink documentation
-# http://mavlink.org/messages/common#ENUM_MAV_CMD
-
-# some common MAV_CMD
-uint16 CMD_DO_SET_MODE = 176
-uint16 CMD_DO_JUMP = 177
-uint16 CMD_DO_CHANGE_SPEED = 178
-uint16 CMD_DO_SET_HOME = 179
-uint16 CMD_DO_SET_RELAY = 181
-uint16 CMD_DO_REPEAT_RELAY = 182
-uint16 CMD_DO_SET_SERVO = 183
-uint16 CMD_DO_REPEAT_SERVO = 184
-uint16 CMD_DO_CONTROL_VIDEO = 200
-uint16 CMD_DO_SET_ROI = 201
-uint16 CMD_DO_MOUNT_CONTROL = 205
-uint16 CMD_DO_SET_CAM_TRIGG_DIST = 206
-uint16 CMD_DO_FENCE_ENABLE = 207
-uint16 CMD_DO_PARACHUTE = 208
-uint16 CMD_DO_INVERTED_FLIGHT = 210
-uint16 CMD_DO_MOUNT_CONTROL_QUAT = 220
-uint16 CMD_PREFLIGHT_CALIBRATION = 241
-uint16 CMD_MISSION_START = 300
-uint16 CMD_COMPONENT_ARM_DISARM = 400
-uint16 CMD_GET_HOME_POSITION = 410
-uint16 CMD_START_RX_PAIR = 500
-uint16 CMD_REQUEST_AUTOPILOT_CAPABILITIES = 520
-uint16 CMD_DO_TRIGGER_CONTROL = 2003
-
-# Waypoint related commands
-uint16 NAV_WAYPOINT = 16
-uint16 NAV_LOITER_UNLIM = 17
-uint16 NAV_LOITER_TURNS = 18
-uint16 NAV_LOITER_TIME = 19
-uint16 NAV_RETURN_TO_LAUNCH = 20
-uint16 NAV_LAND = 21
-uint16 NAV_TAKEOFF = 22
+# http://mavlink.org/messages/common
+
+# [[[cog:
+# from pymavlink.dialects.v20 import common
+# from collections import OrderedDict
+# import re
+#
+# def wr_enum(enum, ename, pfx='', bsz=16):
+#     cog.outl("# " + ename + "_" + pfx)
+#     for k, e in enum:
+#         # exclude also deprecated commands
+#         if 'MAV_CMD' + "_" + pfx in e.name and not re.search('deprecated', e.description, re.IGNORECASE):
+#             sn = e.name[len('MAV_CMD') + 1:]
+#             l = "uint{bsz} {sn} = {k}".format(**locals())
+#             if e.description:
+#                 l += ' ' * (50 - len(l)) + ' # ' + e.description
+#             cog.outl(l)
+#     cog.out('\n')
+#
+# def decl_enum(ename):
+#     enum = sorted(common.enums[ename].items())
+#     enum.pop() # remove ENUM_END
+#
+#     enumt = []
+#     # exception list of commands to not include
+#     exlist = ['SPATIAL', 'USER', 'WAYPOINT']
+#     for k, e in enum:
+#         enumt.extend(e.name[len(ename) + 1:].split('_')[0:1])
+#
+#     enumt = sorted(list(OrderedDict.fromkeys(enumt)))
```

Comment:
What this do?!

Reply:
It removes the duplicates keys from `enumt` and then sorts them alphabetically.